extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationNotifications.js

```diff
@@ -211,6 +215,8 @@ require(
 			},
 
 			wikiClick: function (e) {
+				this.track('wiki');
```

Comment:
It will end up us `trackingevent notifications/click/wiki/` event in GA. I believe it has enough context.